dataflows/processors/dumpers/file_formats.py

```diff
@@ -71,7 +71,7 @@ def __transform_value(self, value, field):
     def write_transformed_row(self, *_):
         raise NotImplementedError()
 
-    def write_row(self, row):
+    def write_row(self, row, schema):
```

Comment:
why do you need to add the schema? it's already available in the `self.fields` dict.

Reply:
You are right, I didn't realize that I could get the field type in ```self.fields```